configure.ac

```diff
@@ -827,14 +827,14 @@ TEMP_CPPFLAGS="$CPPFLAGS"
 CPPFLAGS="$CPPFLAGS $BOOST_CPPFLAGS"
 AC_MSG_CHECKING([for mismatched boost c++11 scoped enums])
 AC_LINK_IFELSE([AC_LANG_PROGRAM([[
-  #include "boost/config.hpp"
-  #include "boost/version.hpp"
+  #include <boost/config.hpp>
```

Comment:
@theuni as this is part of a check, I wonder if there was a specific resaon for using `"` instead of `<`?

Reply:
For reference this code came from #7302